### ISSUE TITLE
config.title and config.description is not rendering

### ISSUE THREAD

**Issue by adrianboston** [Keats/gutenberg#2154]
# Bug Report

## Environment

Zola version:

zola 0.16.1

macOS Darwin Kernel Version 20.6.0

## Expected Behavior

config.toml contains an entry for title and description near the top of the file

```
base_url = "http://127.0.0.1:1111"

title = "My site"
description = "My site description"

```

I’m using a macro for the header which is used within the base.html template. 

```
{% import "macros/head.html" as head %}
{% import "macros/nav.html" as nav %}

<!DOCTYPE html>

<html lang="{{lang}}">

    {{ head::header(config = config, security_config = config.extra.security) }}

```

macros/head.html contains the macro

```
{% macro header(config, security_config) %}
<head>
    ...
    <meta name="description" content="{{ config.description }}" />
    ...
    <meta property="og:description" content="{{ config.description }}" />
```

The expected behavior is transforming that code into 

```
 <meta name="description" content="My site description">
 ...
 <meta property="og:description" content="My site description">
```

## Current Behavior

Instead the description is rendered as an empty string within the rendered/built site -- on every html file whether a section or a page

```
 <meta name="description" content="">
 
 <meta property="og:description" content="">
```

Other variables in config.toml are converted. For example. the following are transformed as expected.

```
<meta name="base" content="{{config.base_url}}"/>
<meta name="copyright" content="{{config.extra.author}}"/>

```
The variables that render as expected that are within the bottom or base section of config.toml are the following two:

base_url = "http://127.0.0.1:1111"
default_language = "en"

Additional variables that do not render held within the base section are the following. Noting that I have no expectation for them to render so thats fine. 

output_dir = "/Volumes/GODZILLA/var/www/zola"
compile_sass = true

Placing title and description under [extra] section of config.toml worked as expected

```
[extra]
title = "My site"
description = "My site description"

```

NOTE that there is no error or panic during the build stage.

## Step to reproduce

Make certain you have a description and title in the config.toml file

Place the following in a macro that is called from a template. See above

    <meta name="description" content="{{ config.description }}" />

Check the browser developer tools

**Comment by Keats** [comment]
You're using `page.description` but not passing  `page` as a parameter to the `header` macro

**Comment by adrianboston** [comment]
Thats a typo. its config. 
Ill edit that

**Comment by proofy** [comment]
I have the same problem, but _config.description_ renders in default language and not in other languages. I also use it in a macro for meta descriptions.

**Comment by proofy** [comment]
oh, I found something: 

```
title = "Super Website"
description = "Best site in the world"

```
you have to repeat this both for every language

```
[languages]
[languages.de]
title = "Super Webseite"
description = "Beste Seite in der Welt"

```

**Comment by proofy** [comment]
I think it would be relevant to specify in the documentation whether the configuration item is language dependent or not

**Comment by adrianboston** [comment]
I am not using localization in this case. just

       <meta name="description" content="{{ config.description }}" />

for localization i would expect it to be very different

      <meta name="description" content="{{ trans(key="description", lang=lang) }}" />

**Comment by proofy** [comment]
Not in the template, in the config.toml. I just only use 
`   <meta name="description" content="{{ config.description }}" />`
in the template and zola use different content direct without trans() if I have localization content with e.g. index.de.md . 

so, what you can test now, if you have defined the default language, remove it, if not, use it with language definition in config.toml . Just play with it a little bit.

**Comment by Keats** [comment]
Yes, configs in templates are localized. In practice in the templates, it means that you need to set title/description for each language if you are using them.
